### DISCUSSION TITLE
create a Patreon or Kofi

### DISCUSSION THREAD

**Discussion by QuetzalcoutlDev** [LMMS/lmms#6744] [General]
### Enhancement Summary
I know you have surely heard this many times, but why doesn't LMMS have a Patreon? or a page in Kofi?

I think it would be a good idea for developers, I know that what LMMS needs is not so much money but developers and I tell them if I were an expert programmer I would get involved in the development of LMMS without thinking about it 😅, although that's not what I was going to say ...

Well I think it would be a good idea to create a Patreon because there are many people who want to support LMMS financially but let's be honest, I think giving and not receiving anything in return is what doesn't motivate them, but not because we are ambitious or bad if not that we want to know that LMMS is still standing and I think that if they give that information on Patreon or Kofi, it would be a good idea because that way LMMS will have a way to maintain itself and the developers too, also a good way to interact directly with its users is that if we are on Patreon allow us to send demos of songs made by us and place them on LMMS

I don't know if I made myself understood 😅
#### Justification
There are people who want to support LMMS and creating a Patreon or Kofi makes it a bit more fun also the donation button on the page is very hidden and I think putting the Patreon or Kofi button next to the download button would be nice

**Comment by RiedleroD** [comment]
I had a patreon until recently and can't recommend it. If anything, liberapay.com is a good alternative.

**Comment by Monospace-V** [comment]
From what I know, I am under the impression that ko-fi is used by content creators, like artists, podcasters, youtubers, etc. I think ko-fi's point is that you donate to pay your content creator, and to unlock special content visible only to donors.
Ko-fi is therefore a little unusual a choice for a software. And making specific content available only to donors on a FOSS app? Doesn't seem like the best idea.
I'd say Patreon is more suited for LMMS, but unlike Riedler I have never used Patreon, so I cannot comment on how it is.

I do agree that the donation link could be a little more obvious.
Even though it's in a fairly sensible place, I've only registered it passively, and realise that I do not have any specific memory of seeing an actual Donate link, unlike other rarely-used parts of the site.

**Comment by QuetzalcoutlDev** [comment]
> From what I know, I am under the impression that ko-fi is used by content creators, like artists, podcasters, youtubers, etc. I think ko-fi's point is that you donate to pay your content creator, and to unlock special content visible only to donors. Ko-fi is therefore a little unusual a choice for a software. And making specific content available only to donors on a FOSS app? Doesn't seem like the best idea. I'd say Patreon is more suited for LMMS, but unlike Riedler I have never used Patreon, so I cannot comment on how it is.
> 
> I do agree that the donation link could be a little more obvious. Even though it's in a fairly sensible place, I've only registered it passively, and realise that I do not have any specific memory of seeing an actual Donate link, unlike other rarely-used parts of the site.

well it was just an idea

So Patreon would be a good option for LMMS, yes I know that Patreon is not that nice but it would be the best option because many currently use Patreon and I have seen that many free software projects use Patreon

But that's up to the developers.

**Comment by kabrownie** [comment]
hello, I think there is a sponsor button available on this repository. at the top
![Screenshot from 2022-10-04 22-15-48](https://user-images.githubusercontent.com/75438336/193906708-128e5b3a-3ce2-4763-a19a-9f29de1034df.png)

**Comment by JohannesLorenz** [comment]
I think offering alternatives like patreon or kofi is good. It might seem redundant, but I guess one reason (aside from "no money") for people to not donate is that they are too lazy. If you have a patreon or kofi account (and these are people who *do* have enough money :grin:), you might just be too lazy to create a PayPal account just for sending 1$ to LMMS every month.

Also, until I checked it for myself, I did not know that paypal allows comfortable monthly donations, so I would not have used it. For patreon, we all know that it allows monthly donations.

All in all, to maximize income, we need to maximize the sources. We can even have 10 or 20 sources...

About what we can offer on patreon: We could just allow levels where you receive nothing in favor (pure donations). Aside, the most valuable we can offer might be our knowledge. Developers could solve their issues in a livestream and explain what they are doing - this could also help us gaining new developers.

**Comment by QuetzalcoutlDev** [comment]
> I think offering alternatives like patreon or kofi is good. It might seem redundant, but I guess one reason (aside from "no money") for people to not donate is that they are too lazy. If you have a patreon or kofi account (and these are people who _do_ have enough money 😁), you might just be too lazy to create a PayPal account just for sending 1$ to LMMS every month.
> 
> Also, until I checked it for myself, I did not know that paypal allows comfortable monthly donations, so I would not have used it. For patreon, we all know that it allows monthly donations.
> 
> All in all, to maximize income, we need to maximize the sources. We can even have 10 or 20 sources...
> 
> About what we can offer on patreon: We could just allow levels where you receive nothing in favor (pure donations). Aside, the most valuable we can offer might be our knowledge. Developers could solve their issues in a livestream and explain what they are doing - this could also help us gaining new developers.

In fact it would be nice to offer insights so I could join the development as I would love to know how LMMS works internally 😄 as I am looking for ways to learn C++ and Qt and help

I think that would attract new developers 😁😁
